src/main/io/serial_msp.h

```diff
@@ -234,8 +234,8 @@ static const char * const boardIdentifier = TARGET_BOARD_IDENTIFIER;
 
 #define MSP_EEPROM_WRITE         250    //in message          no param
 
+#define MSP_DEBUG                252    //out message         debug1,debug2,debug3,debug4
```

Comment:
bad idea.  clients already use this message.  we don't want to force them to change.